packages/devtools_server/lib/src/server.dart

```diff
@@ -233,3 +259,17 @@ void printOutput(
 }) {
   print(machineMode ? jsonEncode(json) : message);
 }
+
+/// Map the URI (which may already be Observatory web app) to a WebSocket URI
```

Comment:
Add a line break after :"from the VM service."
The title of an comment should be on a separate line from following sentences.

Reply:
Done!